packages/cli/src/commands/init/utils.ts

```diff
@@ -358,11 +358,15 @@ export const mastra = new Mastra()
       `
 import { Mastra } from '@mastra/core/mastra';
 import { createLogger } from '@mastra/core/logger';
+import { LibSQLStore } from '@mastra/libsql';
 ${addWorkflow ? `import { weatherWorkflow } from './workflows';` : ''}
 ${addAgent ? `import { weatherAgent } from './agents';` : ''}
 
 export const mastra = new Mastra({
   ${filteredExports.join('\n  ')}
+  storage: new LibSQLStore({
+    url: ":memory:",
```

Comment:
I think this makes more sense to be a local file - otherwise every time code is saved the db state will reset. I think during debugging/testing that would be super annoying. I know folks can change it to that, but I don't think newer devs will know

Reply:
By default this is just for evals and telemetry so I think memory is fine, i can add a comment to it.